cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,61 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+import Long from 'long'
+import chalk from 'chalk'
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    memberId: flags.integer({
+      required: true,
+      description: 'Membership ID of the validator to verify',
+    }),
+    isVerified: flags.boolean({
+      required: true,
```

Comment:
```suggestion
     default: false,
```

We can't specify boolean flags as `isVerified=true` or `isVerified=false`, so the only way to signal that `isVerified` is true/false is by presence/absence of the flag itself, i.e.

```
./bin/run working-groups:verifyValidator --memberId=2 --isVerified # isVerified is 'true'
./bin/run working-groups:verifyValidator --memberId=2                     # isVerified is 'false'
```

Since we would define the default value of `isVerified` false so if we omit the flag then it was be assumed false in the command.